diff -urN flpsed-0.7.0/src/Postscript.cxx flpsed-0.7.0-patched/src/Postscript.cxx
--- flpsed-0.7.0/src/Postscript.cxx	2011-09-21 07:19:06.000000000 +1100
+++ flpsed-0.7.0-patched/src/Postscript.cxx	2012-02-28 19:53:57.000000000 +1100
@@ -311,7 +311,7 @@
 	tag_format   = PS_TAG_FORMAT;
 
 	fprintf(out, "\n");
-	fprintf(out, ps_header());
+	fprintf(out, "%s", ps_header());
 
 	for (int i = 1; i<pse->get_max_pages(); i++) {
 		if (pse->get_text(i)) {
@@ -321,7 +321,7 @@
 		}
 	}
 
-	fprintf(out, ps_trailer());
+	fprintf(out, "%s", ps_trailer());
 	fprintf(out, "\n");
 	fprintf(out, "%s", PSEDIT_END);
 	fprintf(out, "\n");
